src/cryptoadvance/specter/templates/wallet/settings/wallet_settings.jinja

```diff
@@ -180,28 +181,32 @@
 				<div class="row aligned">
 					<textarea name="address_labels_data" 
 					id="address_labels_data" 
-					placeholder="{{ _('You can paste or drag & drop here: Electrum JSON file, Specter addresses CSV or Specter JSON file.') }}"  
-					ondrop="dropHandler(event, 'address_labels_data');" 
-					ondragover="dragOverHandler(event, 'address_labels_data');"
-					ondragleave="dragLeaveHandler(event, 'address_labels_data');" 
-					ondragexit="dragExitHandler(event, 'address_labels_data');" 
-					ondragend="dragEndHandler(event, 'address_labels_data');"></textarea>
+					placeholder="{{ _('You can paste and drag & drop here') }}
+					{{ _('(Electrum JSON file, Specter addresses CSV or Specter JSON file)') }}"  
```

Comment:
There are spaces hidden here, that result in:
![image](https://user-images.githubusercontent.com/60378539/165623204-f558f6d5-8f23-4c3c-96b9-ff351043239f.png)

Reply:
I've used this approach here now:
https://stackoverflow.com/questions/7312623/insert-line-break-inside-placeholder-attribute-of-a-textarea

Looks find for me now:

![image](https://user-images.githubusercontent.com/70536101/167099790-ef6a0b77-f56a-4b3a-b353-a94fec5a15f9.png)